Issue2

damonhuang233 opened the pull request.

damonhuang233 committed:
Able to add ship to do: check vaild and add occuped to ship

damonhuang233 committed:
Able to add ship to do: check vaild and add occuped to ship

damonhuang233 committed:
add vaild check

damonhuang233 committed:
fix index

damonhuang233 committed:
fix index

Carlos231 commented on src/main/java/cs361/battleships/models/Ship.java:
```diff
@@ -7,18 +7,42 @@
+		this.occupiedSquares = occupiedSquares;
+	}
+
+  public String getKind(){
```
need an indent here

Carlos231 commented on src/main/java/cs361/battleships/models/Ship.java:
```diff
@@ -7,18 +7,42 @@
+		this.occupiedSquares = occupiedSquares;
+	}
+
+  public String getKind(){
```
could use an indent here

damonhuang233 replied:
What's an indent?

kryolight replied:
An indent is a tab. I decided to add the tab in since it just makes the code look nicer.

kryolight reviewed (approved):
Everything looks good. It's commented well and cleanly written.

kryolight committed:
Merge branch 'master' into issue2

kryolight committed:
Added a tab to Ship.java

Added a tab to make code look nicer. The addition should have no effect on the code.
